solidity/contracts/Coordinator.sol

```diff
@@ -320,6 +308,36 @@ contract Coordinator is ChainlinkRequestInterface, CoordinatorInterface {
     return keccak256(abi.encodePacked(_payment, _expiration, _endAt, _oracles, _requestDigest));
   }
 
+  /**
+   * @dev Stores an oracle's response and then increments the total number of repsonse received.
+   * @param _requestId is the unique identifier generated by the requester and their nonce
+   * @param _data is the actual answer to a request submitted by an oracle node
+   */
+  function storeResponse(bytes32 _requestId, bytes32 _data) private {
+    callbacks[_requestId].responses[msg.sender] = uint256(_data);
+    callbacks[_requestId].responseCount += 1;
+  }
+
+  /**
+   * @dev Aggregates the responses in storage and cleans them up.
+   * Then pays associated oracles their share of LINK.
+   * Deletes the request's callback record, and finally returns the aggregated result.
+   * @param _requestId is the unique identifier generated by the requester and their nonce
+   * @param _paymentAmount is the amount of LINK shared between the oracles
+   * @param _oracles is the list of oracle addresses
+   */
+  function aggregateAndPay(bytes32 _requestId, uint256 paymentAmount, address[] memory oracles) private returns (uint256) {
```

Comment:
Can't do suggestion, `paymentAmount` should be `_paymentAmount` and `oracles` should be `_oracles`.